ext/dba/dba.stub.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/** @return resource|false */
+function dba_popen($path, $mode, $handlername = UNKNOWN, ...$handler_parameters) {}
+
+/** @return resource|false */
+function dba_open($path, $mode, $handlername = UNKNOWN, ...$handler_parameters) {}
+
+/** @param resource $handle */
+function dba_close($handle): void {}
+
+/**
+ * @param string|array $key
+ * @param resource $handle
+ */
+function dba_exists($key, $handle): bool {}
+
+/**
+ * @param string|array $key
+ * @param resource $handle
+ * @return string|false
+ */
+function dba_fetch($key, $skip, $handle = UNKOWN) {}
```

Comment:
I realise this one is all wacky because of the variable args, but $skip is a legitimate `int` type when it's handled, isn't it?

Reply:
When `$skip` is omitted, the `$handle` is passed as second argument, so the second argument can either be `int` or `resource`. Obviously, I have forgotten to document that as well.